app/src/main/java/com/example/myapplication/FirstFragment.java

```diff
@@ -29,23 +29,50 @@ public View onCreateView(
         userModel.setMatchData(matchData);
         binding = FragmentFirstBinding.inflate(inflater, container, false);
         v = container;
+<<<<<<< HEAD
```

Comment:
there's some unresolved merge conflicts here that care causing the build tests to fail